libs/common/src/assets/i18n/en.json

```diff
@@ -1645,6 +1645,10 @@
 	"tasks-automation.actions.delete-data.description": "The generated data in your Knowledge Box will be permanently deleted, and the active jobs will be stopped. You can continue modifying your agent after.",
 	"tasks-automation.actions.delete-data.success": "Your generated data will be deleted shortly.",
 	"tasks-automation.actions.delete-data.title": "Delete generated data",
+	"tasks-automation.actions.duplicate.action": "Duplicate",
+	"tasks-automation.actions.duplicate.description": "A new agent will be created based on the same parameters as the original agent. if the agent defines a field name, it will be complted with the \"_copy\" suffix, for unicity sakes.",
```

Comment:
```suggestion
	"tasks-automation.actions.duplicate.description": "A new agent will be created based on the same parameters as the original agent. If the agent defines a field name, it will be completed with the \"_copy\" suffix, for unicity sakes.",
```

Reply:
I don't understand very well your explanations: I saw the dialog has a field "Agent name", so if I understand well, the user duplicate an agent and have to choose a name for the new agent resulting from this duplication, right? Now I don't get where this `_copy` suffix will be added.